MoltenVK/MoltenVK/GPUObjects/MVKDevice.mm

```diff
@@ -1243,6 +1243,7 @@
 #if MVK_MACOS
     if (mvkOSVersionIsAtLeast(10.14)) {
         static const uint32_t kAMDVendorId = 0x1002;
+        initGPUInfoProperties();    // Need to know the vendor ID for this.
```

Comment:
Hmmm...calling `initGPUProperties()` twice, with one of them out-of-place, rubs me the wrong way.

But alternates that I can think of, such as renaming `initGPUProperties()` to `retrieveGPUInfo()`, having it store the values in separate member variables and then copy them into `_properties` within `initProperties()` are also less than optimal and would be intrusive at this point in the dev cycle given there are three implementations of `initGPUProperties()`.

I might put that step on my TODO list at some point.